addons/website_slides/models/slide_channel.py

```diff
@@ -99,9 +99,12 @@ def _default_access_token(self):
     tag_ids = fields.Many2many(
         'slide.channel.tag', 'slide_channel_tag_rel', 'channel_id', 'tag_id',
         string='Tags', help='Used to categorize and filter displayed channels/courses')
-    category_ids = fields.One2many('slide.category', 'channel_id', string="Categories")
     # slides: promote, statistics
-    slide_ids = fields.One2many('slide.slide', 'channel_id', string="Slides")
+
```

Comment:
HumanLinter: remove blank lines in fields declaration, we try to keep it a bit compacted :)